docs/data/migration/migration-pickers-v6/migration-pickers-v6.md

```diff
@@ -410,6 +420,34 @@ The `getYearRange` method used to accept two params and now accepts a tuple to b
 + adapter.getYearRange([start, end])
 ```
 
+### Restrict the input format of the `date` method
+
+The `date` method now have the behavior of the v6 `dateWithTimezone` method.
+The following changes have been applied:
+
+- Its first argument is now mandatory, you need to pass `undefined` to get a date with the current timestamp instead of passing no argument
+
+  ```diff
+  - adapter.date();
+  + adapter.date();
+  ```
+
```

Comment:
```suggestion
```

Not sure it's usefull to replace `adapter.date()` by `adapter.date()` XD

Reply:
:laughing: Search and replace works well but do not remove useless doc sections